roles/icingaweb2/tasks/manage_icingaweb_config.yml

```
@@ -0,0 +1,139 @@
+- name: Set permissions on icingaweb config
+  ansible.builtin.file:
+    state: directory
+    dest: "{{ icingaweb2_config_dir }}"
+    owner: "{{ icingaweb2_httpd_user }}"
+    group: "{{ icingaweb2_group }}"
+    mode: "2770"
+
+- name: Set permissions on enabledModules
+  ansible.builtin.file:
+    state: directory
+    dest: "{{ icingaweb2_config_dir }}/enabledModules"
+    owner: "{{ icingaweb2_httpd_user }}"
+    group: "{{ icingaweb2_group }}"
+    mode: "2770"
+
+- name: Set permissions on enabledModules
```

```suggestion
- name: Set permissions on modules
```

This should be `modules` instead of `enabledModules`, right?